core/pom.xml

```diff
@@ -174,6 +174,14 @@
             <scope>test</scope>
         </dependency>
 
+        <!-- https://mvnrepository.com/artifact/com.google.code.gson/gson -->
+        <dependency>
+            <groupId>com.google.code.gson</groupId>
```

Comment:
If this library is only used serialization matters for JSONs, then you can use the already imported ObjectMapper from com.fasterxml.jackson.databind and remove this dependency.

Reply:
Yes, it was only for JSON serialization. Will be removed and replaced with ObjectMapper.